frontend/packages/ux-editor/src/data/formItemConfig.ts

```diff
@@ -1,11 +1,32 @@
 import { ComponentType } from 'app-shared/types/ComponentType';
 import { FormItem } from '../types/FormItem';
 import { FormPanelVariant } from '../types/FormComponent';
+import {
+  InformationIcon,
+  FingerButtonIcon,
+  FileTextIcon,
+  PaperclipIcon,
+  CheckmarkIcon,
+  CalendarIcon,
+  ChevronDownIcon,
+  FileCodeIcon,
+  TasklistIcon,
+  PinIcon,
+  ExclamationmarkIcon,
+  MenuHamburgerIcon,
+  EyeClosedIcon,
+  TabsIcon,
+  TableIcon,
+  CaptionsIcon,
+  BulletListIcon,
+  BookmarkIcon,
+  ChevronDownDoubleIcon,
+} from '@navikt/aksel-icons';
 
 export type FormItemConfig<T extends ComponentType = ComponentType> = {
   name: T;
   defaultProperties: FormItem<T>;
-  icon?: string;
+  icon?: string | React.ComponentType;
```

Comment:
Fjern `string` og sett inn props for å gjøre det mulig å sette `title` på ikonet. Ser at Aksel har trikset litt for å få inn `title`-propen, men de eksporterer ikke den typen, så vi må nesten hardkode den:
```suggestion
  icon?: React.ComponentType<SVGProps<SVGElement> & { title: string }>;
```
Du må vel også importere `React` for at dette skal fungere?

Reply:
den funket ikke , blir feil alle ikoner fordi type er endret.

Reply:
Ok, må vel kopiere hele typen fra Aksel da:
`React.ForwardRefExoticComponent<React.ComponentType<SVGProps<SVGSVGElement> & { title: string; titleId: string }> & RefAttributes<SVGSVGElement>>`